src/routes/forms.js

```diff
@@ -72,6 +78,35 @@ export default [
       }
     }
   },
+  {
+    method: 'PATCH',
+    path: '/forms/{id}',
+    /**
+     * @param {RequestFormMetadataUpdateById} request
+     */
+    async handler(request) {
+      const { params, payload } = request
+      const { id } = params
+
+      const slug = await updateFormMetadata(id, payload)
+      logger.info(`Updated form metadata (draft) for form ID ${id}`)
+
+      return {
+        id,
+        slug,
+        status: 'updated'
+      }
+    },
+    options: {
+      validate: {
+        params: formByIdSchema,
+        payload: formMetadataInputSchema.fork(
```

Comment:
worth a comment here stating why you're doing this

e.g.

"Take the form metadata update schema and make all fields optional. This acts similar to `Partial<T>` in Typescript."